packages/compile-solidity-tests/package.json

```diff
@@ -35,7 +35,7 @@
     "fs-extra": "^9.1.0",
     "mocha": "10.1.0",
     "sinon": "^9.0.2",
-    "solc": "0.8.18",
+    "solc": "0.8.19",
```

Comment:
Nice, you caught this new addition!

Reply:
I have used `npx lerna-update-wizard` to upgrade the **solc** version and noticed that there was this new addition of solc in the `compile-solidity-tests` package. :)